src/dataviews/dataview-model-base.js

```diff
@@ -120,6 +129,12 @@ module.exports = Model.extend({
     }
   },
 
+  _initialFetch: function () {
+    this.fetch({
+      success: this._onChangeBinds.bind(this)
```

Comment:
We were not handling errors before. I just extracted this to a method so that I was able to reuse it. Having said this, I notice that in the following `fetch`s, we set the state of the dataview and trigger an `error` event if request fails ([here](https://github.com/CartoDB/cartodb.js/blob/df5ec1cd78ef442bb463ee4cea137ae3e6a474f0/src/dataviews/dataview-model-base.js#L302-L312)). We could do that in this initial fetch too...